src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/products/products.yml

```diff
@@ -31,20 +31,11 @@ _product_warehouses:
 
 # Product List
 admin_product_catalog:
-    path: /{offset}/{limit}/{orderBy}/{sortOrder}
+    path: /
     methods:  [GET, POST]
     defaults:
         _controller: PrestaShopBundle:Admin/Product:catalog
         _legacy_controller: AdminProducts
-        limit: last
-        offset: 0
-        orderBy: last
-        sortOrder: last
-    requirements:
-        limit: _limit|last|\d+
-        orderBy: last|id_product|name|reference|name_category|price|sav_quantity|position|active|position_ordering
```

Comment:
we're losing every "value" validation here

Reply:
+1